android/src/main/java/com/mybigday/rns3/RNS3TransferUtility.java

```diff
@@ -94,19 +95,25 @@ public void onStateChanged(int id, TransferState state) {
         TransferObserver task = transferUtility.getTransferById(id);
         WritableMap result = Arguments.createMap();
         result.putMap("task", convertTransferObserver(task));
-        sendEvent("@_RNS3_Events", result);
+        getReactApplicationContext()
+                .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
+                .emit("State_Changed", result);
```

Comment:
Just `sendEvent("State_Changed", result)`? There are different?

Also, I prefer to use `@RNS3_` as a event name prefix, we must ensure it don't conflict with other libraries.

Also,